logger/log.go

```diff
@@ -64,8 +64,10 @@ func Set(level string) {
 		zerolog.SetGlobalLevel(zerolog.InfoLevel)
 	case "debug":
 		zerolog.SetGlobalLevel(zerolog.DebugLevel)
+		Debug = true
```

Comment:
Using this variable has disadvantages. Settings this to true also generates a log of log files that are currently only set via `DEBUG=1`. Can we instead change `isatty.IsTerminal(os.Stdout.Fd()) && !logger.Debug` to something that checks the log-level?

Reply:
I exposed the log level and check for it.